Formula/svtplay-dl.rb

```diff
@@ -3,8 +3,8 @@ class SvtplayDl < Formula
 
   desc "Download videos from https://www.svtplay.se/"
   homepage "https://svtplay-dl.se/"
-  url "https://files.pythonhosted.org/packages/b8/f3/00b9a5cbd1f8f759ac8bdcb632114aab622db8eb3c8702a48ef166acbfa3/svtplay-dl-2.5.tar.gz"
-  sha256 "341eb21c445a5779bef6659406d21519561bb83a4784155cc617d3c1c0995afa"
+  url "https://files.pythonhosted.org/packages/f2/7d/d78f5bfb5578bd5c936aa61e8e4b2b9346d2315e45af6a0e6af76dd48882/svtplay-dl-2.6.tar.gz"
+  sha256 "8287042f6f46b1e38d11882df5a2620461e30e11a74e6355857c586b4211a860"
 
```

Comment:
Could you add the license line as below:

```suggestion
  license "MIT"
  
```

Reply:
sure :)